partiql-parser/src/main/kotlin/org/partiql/parser/internal/PartiQLParserDefault.kt

```diff
@@ -2022,6 +2023,12 @@ internal class PartiQLParserDefault : PartiQLParser {
          */
 
         override fun visitBag(ctx: GeneratedParser.BagContext) = translate(ctx) {
+            // Prohibit hidden characters between angle brackets
+            val startTokenIndex = ctx.start.tokenIndex
+            val endTokenIndex = ctx.stop.tokenIndex
+            if (tokens.getHiddenTokensToRight(startTokenIndex, GeneratedLexer.HIDDEN) != null || tokens.getHiddenTokensToLeft(endTokenIndex, GeneratedLexer.HIDDEN) != null) {
+                throw error(ctx, "Invalid bag expression")
+            }
```

Comment:
I think we should also port the fix to the some of the binary ops. 

https://github.com/partiql/partiql-lang-kotlin/blob/4dd09721d56cacf19f4b7e312ac927f247a20975/partiql-parser/src/main/kotlin/org/partiql/parser/internal/PartiQLParserDefault.kt#L1616-L1624

But feel free to leave this out for now.

Reply:
Yeah good catch, for `<>` and `!=` this is also an issue. In the operator PR https://github.com/partiql/partiql-lang-kotlin/pull/1499, I'll fix the issues there since there is a refactoring of the special symbol parsing.